File: C_Programming/Day_4/program16.c
// Description : Write a program that accepts number from the user &  
//               check whether that number is prime or not

#include<stdio.h>
#include<stdbool.h>

bool CheckPrime(int iNo)
{
    int iCnt = 0;
    bool bFlag = true;

    for(iCnt = 2 ; iCnt <= iNo/2 ; iCnt++)
    {
        if(iNo % iCnt == 0)
        {
            bFlag = false;
        }
    }

    return bFlag;

}

int main()
{
    int iValue = 0;
    bool bRet = false;

    printf("Enter the number : ");
    scanf("%d",&iValue);

    bRet = CheckPrime(iValue);

    if(bRet == true)
    {
        printf("It's an prime number...");
    }
    else
    {
        printf("It's not an prime number...");
    }

    return 0;
}

File: Python_Programming/Day_4/program16.py
# Description : Write a program that accepts number from the user &  
#               check whether that number is prime or not

def CheckPrime(No):
    Flag = True

    for i in range(2 , No//2 , 1):
        if(No % i == 0):
            Flag = False

    return Flag

def main():

    Value = 0
    Ret = False

    print("Enter the number : ")
    Value = int(input())

    Ret = CheckPrime(Value)

    if(Ret == True):
        print("It's an prime number...")
    else:
        print("It's not an prime number...")


if __name__ == "__main__":
    main()